cranelift/codegen/src/isa/x64/inst/emit.rs

```diff
@@ -1787,6 +1787,40 @@ pub(crate) fn emit(
             sink.put1(*imm)
         }
 
+        Inst::XmmLoadConst { val, dst, ty } => {
+            // This sequence is *one* instruction in the vcode, and is expanded only here at
+            // emission time, because we cannot allow the regalloc to insert spills/reloads in
+            // the middle; we depend on hardcoded PC-rel addressing below. TODO Eventually this
```

Comment:
My convention has been to put `Seq` in the name for such synthetic instruction sequences, can you add the suffix too, please?